### PR TITLE
Add missing skill constants

### LINKED ISSUE #42: Constants for class skills are missing some entries
I've found that a few classes are missing entries for skills in their java constants file (`constants.skills.*`). Two examples are Aran and Bowmaster. 

If you cross check the skills listed against those available on BBB hiddenstreet, you can which are missing. 

They'd be useful to add in so that they could be referenced directly instead of the skill id, which would improve script readability in my opinion. Of course, it's better to have a complete list in case people wish to reference them in future, so that's another reason.

### PR DESCRIPTION
**Pull request by cpurules**
Resolves #42 

No existing skill ID values were touched, so this is a non-breaking change.

Cross referenced the BBB Hidden-Street website with the skill handbook included.  For KoC and Aran, also reordered/grouped the skills based on job advancement level.

Did not touch Evan - I have never played an Evan and have no idea what's going on with the multiple mastery levels :) So some skills may be missing there.

### CODE REVIEW

File: src/main/java/constants/skills/Bowmaster.java [hunk 1]
```diff
@@ -26,7 +26,9 @@
  */
 public class Bowmaster {
     public static final int MAPLE_WARRIOR = 3121000;
+    public static final int VENGEANCE = 3121001;
```
Comment (P0nk) [inline src/main/java/constants/skills/Bowmaster.java:29]: "Vengeance" is no real bowmaster skill

File: src/main/java/constants/skills/Crusader.java [hunk 2]
```diff
@@ -25,6 +25,8 @@
  * @author BubblesDev
  */
 public class Crusader {
+    public static final int IMPROVING_MPREC = 1110000;
+    public static final int SHIELD_MASTERY = 1111001;
```
Comment (P0nk) [inline src/main/java/constants/skills/Crusader.java:29]: Should be 1110001

File: src/main/java/constants/skills/WhiteKnight.java [hunk 3]
```diff
@@ -26,6 +26,7 @@
  */
 public class WhiteKnight {
     public static final int IMPROVING_MP_RECOVERY = 1210000;
+    public static final int SHIELD_MASTERY = 1211001;
```
Comment (P0nk) [inline src/main/java/constants/skills/WhiteKnight.java:29]: Should be 1210001

**Review by P0nk — CHANGES_REQUESTED** [review]
Thanks for adding this. It's not immediately useful in the code, but useful as a reference when working with skills. 
A few corrections are needed before I can merge.

**Comment by P0nk** [comment]
As for Evan - it is not part of v83, so it's ok to leave out.

File: src/main/java/constants/skills/Bowmaster.java (hunk 1 above)
Comment (cpurules) [reply src/main/java/constants/skills/Bowmaster.java]: It does show up on BBB but the background is a light red - wonder if it's a dummy skill that isn't actually used.  Removed anyway.

**Review by P0nk — APPROVED** [review]
Looks like a bug on bbb.hidden-street.net. Vengeance might be a skill from later versions.

**Pull request merged**